doc/templates.md

```diff
@@ -100,7 +100,8 @@ The following keys / variables are filled for each datatype
 | `forward_declarations`        | The forward declarations for the user facing classes header files. This is a nested dict, where the keys are namespaces and the leaf values are classes. |
 | `forward_declarations_obj`     | The forward declarations for the `Obj` classes header files.                                                                                             |
 | `is_pod`                      | Flag value indicating whether the `Data` class of this datatype is a POD or if it contains an STL member                                                 |
-| `obj_needs_destructor`        | Flag indicating whether the `Obj` class needs a dedicated destructor for the datatype or whether `= default` is enough.                                  |
+| `obj_needs_destructor`        | Flag indicating whether the `Obj` class needs a dedicated destructor for the datatype or whether `= default` is enough.          
```

Comment:
```suggestion
```

This has been removed from the code, so we can also remove it from the documentation.